### ISSUE TITLE
Rename Attendant(s) to Attendee(s)

### ISSUE THREAD

**Issue by Be-Like** [Tisn/tisn.app#346]
**Describe the bug**
To give better clarity, all references to attendant(s) should be changed to attendee(s)

**Expected behavior**
Provide better clarity

**Additional context**
Error in translation.

**Comment by AlbertoPdRF** [comment]
Thanks for noticing this and opening the issue @Be-Like! This one will be easy to fix, although it will require changes in many places :smile:

**Comment by Be-Like** [comment]
I'd recommend putting the `good first issue` and `hacktoberfest` labels on there as well.

**Comment by AlbertoPdRF** [comment]
You are totally right, I just added those! I'll also do a pass through all open issues to add those labels where applicable :eyes:

**Comment by vvkpd** [comment]
I would like work on this card, could you please assign to me?
Thanks

**Comment by AlbertoPdRF** [comment]
Done @vvkpd! :heavy_check_mark: If you need help with anything, just let me know! Also, we have a Slack workspace that you can join if you want, the invite link is on the README :slightly_smiling_face:

**Comment by kailinwei** [comment]
I would like to work on this as my first time contribution to github, could you please assign to me?

**Comment by AlbertoPdRF** [comment]
Welcome to this repo @kailinwei! :wave:

@vvkpd, are you ok with me assigning this issue to @kailinwei, since they've submitted a PR already? However, @kailinwei, next time please wait for confirmation before starting to work on an issue, as having two people working on the exact same fix can be very frustrating

**Comment by AnmolBansalDEV** [comment]
can I work on this issue? i haven't got any PR merged yet and its the end of hacktoberfest.
